dspace-api/src/test/data/dspaceFolder/config/submission-forms.xml

```diff
@@ -48,18 +48,22 @@
    </form>      
    
    <form name="traditionalpageone">
-     <row>
-       <field>
-         <dc-schema>dc</dc-schema>
-         <dc-element>contributor</dc-element>
-         <dc-qualifier>author</dc-qualifier>
-         <repeatable>true</repeatable>
-         <label>Authors</label>
-         <input-type>name</input-type>
-         <hint>Enter the names of the authors of this item.</hint>
-         <required></required>
-       </field>
-     </row>
+            <row>
+                <relation-field>
+                    <relationshiptype>isAuthorOfPublication</relationshiptype>
+                    <search-configuration>personConfiguration</search-configuration>
+                    <repeatable>true</repeatable>
+                    <label>Author</label>
+                    <hint>Add an author</hint>
+                    <linked-metadata-field>
+                        <dc-schema>dc</dc-schema>
+                        <dc-element>contributor</dc-element>
+                        <dc-qualifier>author</dc-qualifier>
```

Comment:
I didn't want to introduce this right now as it may further complicate the upgrade script for the submission forms
It may be better to revisit it when https://jira.duraspace.org/browse/DS-4046 is being worked on. Would you want me to report it in that ticket?

Reply:
@benbosman : Yes, I'm fine with moving this to that ticket.  I only called it out here cause we are now replicating these same misnamed `dc-*` tags to new locations.  But, I'm fine with leaving this change to a different PR.